tests/test.cpp

```diff
@@ -151,8 +151,37 @@ flatbuffers::unique_ptr_t CreateFlatBufferTest(std::string &buffer) {
 void AccessFlatBufferTest(const uint8_t *flatbuf, size_t length) {
 
   // First, verify the buffers integrity (optional)
-  flatbuffers::Verifier verifier(flatbuf, length);
-  TEST_EQ(VerifyMonsterBuffer(verifier), true);
+  std::vector<uint8_t> test_buff;
+  test_buff.resize(length * 4 + 5);
```

Comment:
This is exactly what I'm doing, first two tests are checking two flatbuffer one after another, but I also wanted to cover non-aligned data, that's why the 3rd and 4th tests are shifted 1 and 2  bytes,  which are resulting into unaligned data, these are needed to cover cases when the flatbuffer data is not aligned in memory.